### PR TITLE
H1 Class/Style for Hero and Subheading  and (SEO) Skipped level headings 

### PR DESCRIPTION
**Pull request by nshonda**
In order to maintain the style for the hero heading and subheadings and fix the skipped level heading issues I had to do the adjustments on the hero component also

related airtable record: https://airtable.com/appsEN1RgrUk3oyon/tbluJ0kJjMnu1rtsx/viwEjc1BEgM0pj72s/rec89EvbBlIuEqDN3?blocks=hide

SEO heading example record: https://airtable.com/appsEN1RgrUk3oyon/tbluJ0kJjMnu1rtsx/viwEjc1BEgM0pj72s/recEQl9Vlm190dWIp?blocks=hide

### CODE REVIEW

File: src/components/organisms/Hero/index.js [hunk 1]
```diff
@@ -113,14 +113,21 @@ function Hero({
           <motion.aside {...animation?.item}>
             {cloneElement(title, {
               sx: {
-                ...(hasTwoCols ? STYLING.headline : STYLING.hero),
+                ...STYLING.hero,
                 fontWeight: ["regular", "light"],
+                color: "blue_500",
```
Comment (quaardvark) [inline src/components/organisms/Hero/index.js:118]: How does this affect pages with blue background Hero?

Comment (nshonda) [reply src/components/organisms/Hero/index.js:118]: hm, I haven't thought about that, we have one page on figma with a blue background: https://www.figma.com/file/YcZAAqtHklQTUQQQWbhNED/Marketing-Pages-Spec?node-id=1%3A712, where both the title and description have a white color, I'm gonna confirm with vini if we should follow this for blue background @quaardvark

Comment (quaardvark) [reply src/components/organisms/Hero/index.js:118]: We have most of the paid search pages with blue hero background as well   - https://info.avail.co/info/listings-100001

**Comment by quaardvark** [comment]
@nshonda can we make sure build is successful?

**Review by quaardvark — APPROVED** [review]

**Pull request merged**